### ISSUE TITLE
Concatenate and minify all the CSSes

### ISSUE THREAD

**Issue by pdehaan** [mozilla/idea-town#62]
We're currently loading some minified Sass and normalize.css and tabzilla.css (plus a xdomain fira.css currently).

It may be worth considering concatenating the 3 CSS files into a single request. Or if this is a temporary site, maybe not. Whatever. It was an interesting problem.

```js
// Styles
gulp.task('styles', function () {
  var cssFiles = [
    'node_modules/normalize.css/normalize.css',
    'node_modules/mozilla-tabzilla/css/*.css',
    'src/styles/main.scss'
  ];
  return gulp.src(cssFiles)
    .pipe(sass().on('error', sass.logError))
    .pipe(autoprefixer('last 2 version'))
    .pipe(concat('main.css'))
    .pipe(gulp.dest('public/styles'))
    .pipe(rename({ suffix: '.min' }))
    .pipe(minifycss())
    .pipe(gulp.dest('public/styles'))
    .pipe(notify({ message: 'Styles task complete' }));
});
```

That should include all the CSS files in the same order as they are currently and then minify them. We can now kill 2 of the gulp tasks for `npm:tabzilla:css` and `npm:normalize`, but probably still need to do something to handle the path to the tabzilla PNG files for the mozilla logo. Or actually, we may not need to worry about that yet since the CSS is hardcoded to `/media` and we have a workaround, but when a new version gets deployed that uses relative `../media` path instead of absolute `/media` path, we may need to rework our workaround.

**Comment by johngruen** [comment]
Fixed by #58

**Comment by johngruen** [comment]
@pdehaan i think this is closed
